apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -306,9 +323,8 @@ Func CameraPipe::build() {
 
     Func denoised = hot_pixel_suppression(shifted);
     Func deinterleaved = deinterleave(denoised);
-    Func demosaiced; Scheduler demosaiced_scheduler;
-    std::tie(demosaiced, demosaiced_scheduler) = demosaic(deinterleaved);
-    Func corrected = color_correct(demosaiced);
+    auto demosaiced = Demosaic::apply(*this, deinterleaved);
```

Comment:
So that approach is easy enough to do, but only works if you are calling from inside another Generator; use from the JIT (as in the inlined_generator test) still requires an explicit context to be passed. Is it worth it to have two very-similar-but-not-quite calls?